### DISCUSSION TITLE
info about pi pico clock

### DISCUSSION THREAD

**Discussion by emilen72** [markadev/AppleII-VGA#55] [General]
Hi, I have a simply curiosity about the clock of the pico on this project... can you tell me how much Mhz is?

thanks

Emiliano

**Comment by markadev** [comment]
Hey there, sure the Pico runs at 126 MHz on the VGA board.  This value was chosen because it is the largest supported speed that is also an integer multiple of the VGA signal pixel clock (25.2 MHz).
